test/fixtures/app/gulpfile.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+var gulp = require('gulp');
+var clean = require('gulp-rimraf');
+var statici18n = require('../../../');
```

Comment:
Ok, I think doing `require('../../../index.js');` might make this a bit more clear.